website/src/components/Screengrabs/atmos-terraform-migrate--help.html

```
@@ -0,0 +1,151 @@
+
+<span style="color:#ff8700;"> </span><span style="color:#ff8700;">█</span><span style="color:#ff8700;">█</span><span style="color:#ff8700;">█</span><span style="color:#ff875f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;"> </span><span style="color:#ff5f5f;"> </span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f5f;">█</span><span style="color:#ff5f87;"> </span><span style="color:#ff0087;">█</span><span style="color:#ff0087;">█</span><span style="color:#ff0087;">█</span><span style="color:#ff0087;"> </span><span style="color:#ff0087;"> </span><span style="color:#ff0087;"> </span><span style="color:#ff0087;"> </span><span style="color:#ff0087;">█</span><span style="color:#ff0087;">█</span><span style="color:#ff00af;">█</span><span style="color:#ff00af;"> </span><span style="color:#ff00af;"> </span><span style="color:#ff00af;">█</span><span style="color:#ff00af;">█</span><span style="color:#ff00af;">█</span><span style="color:#ff00af;">█</span><span style="color:#ff00af;">█</span><span style="color:#ff00af;">█</span><span style="color:#ff00af;"> </span><span style="color:#d700af;"> </span><span style="color:#d700d7;">█</span><span style="color:#d700d7;">█</span><span style="color:#d700d7;">█</span><span style="color:#d700d7;">█</span><span style="color:#d700d7;">█</span><span style="color:#d700d7;">█</span><span style="color:#d700d7;">█</span><span style="color:#d700d7;"> </span>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)